Update README.md (#365)
Appboy/appboy-ios-sdk: README.md

### ⚠ The new [Braze Swift SDK](https://github.com/braze-inc/braze-swift-sdk) is now available! ⚠

> The Appboy iOS SDK (Objective-C SDK) is no longer in active development. All new iOS features Braze releases, like Live Activities and Feature Flags, will only be supported on the [Swift SDK](https://github.com/braze-inc/braze-swift-sdk). Any further updates the Appboy iOS SDK receives will only address security or critical bugs. Beginning March 1, 2024, any remaining support will end, and it will then receive no further updates at all.

Reviewer: ```suggestion
The Appboy iOS SDK (Objective-C SDK) is no longer in active development. All new iOS features Braze releases, like Live Activities and Feature Flags, will only be supported on the [Swift SDK](https://github.com/braze-inc/braze-swift-sdk). Any further updates the Appboy iOS SDK receives will only address security or critical bugs. After  March 1, 2024 this repository will no longer be maintained.
```
